### PR TITLE
Correct "Polishing Rubies" link

### PR DESCRIPTION
**Pull request by SuriyaaKudoIsc**
@drbrain @ffmike @acacheung @qrush @dwradcliffe:

Hi guys, :smile:

I **correct the link** to *http://intridea.com/blog/tag/polishing%20rubies* because the link doesn't exist anymore!

*Yours,*
[**Suriyaa Kudo**](https://github.com/SuriyaaKudoIsc) :octocat:

### CODE REVIEW

File: resources.md [hunk 1]
```diff
@@ -18,7 +18,6 @@ Tutorials
 * [Gemcutter & Jeweler](http://railscasts.com/episodes/183-gemcutter-jeweler)
 * [MicroGems: five minute RubyGems](http://jeffkreeftmeijer.com/2011/microgems-five-minute-rubygems/) - Gems so small that you can store them in a gist.
 * [Let's Write a Gem: Part 1](http://rakeroutes.com/blog/lets-write-a-gem-part-one/) and [Part 2](http://rakeroutes.com/blog/lets-write-a-gem-part-two/)
-* [Polishing Rubies](http://intridea.com/blog/tag/polishing%20rubies)
```
Comment (bf4) [inline resources.md]: Perhaps replace with http://wayback.archive.org/web/20151009055337/http://www.intridea.com/blog/tag/polishing%20rubies ?

**Comment by SuriyaaKudoIsc** [comment]
@bf4 :+1: Done.

**Comment by SuriyaaKudoIsc** [comment]
@drbrain @ffmike @acacheung @qrush: Can be merged.

File: resources.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Tutorials
...
+* [Polishing Rubies](http://wayback.archive.org/web/20151009055337/http://www.intridea.com/blog/tag/polishing%20rubies)
```
Comment (dwradcliffe) [inline resources.md]: Can you make this https?

Comment (SuriyaaKudoIsc) [inline resources.md]: For www.intridea.com ok.
For wayback.archive.org doesn't work.

**Comment by SuriyaaKudoIsc** [comment]
@dwradcliffe: Can be merged. :+1:

File: resources.md (hunk 2 above)
Comment (dwradcliffe) [inline resources.md]: https for wayback worked for me.
polishing rubies intridea blog 2015-12-08 11-34-08

Comment (SuriyaaKudoIsc) [inline resources.md]: OMG! Added it.

**Comment by dwradcliffe** [comment]
Can you squash these 4 commits down to 1 commit?

**Comment by bf4** [comment]
So many comments... :) thanks for this!

B mobile phone

**Pull request closed without merging**